skt/runner.py

```diff
@@ -133,6 +133,13 @@ def __load_blacklist(cls, filepath):
         logging.info('Blacklisted hostnames: %s', hostnames)
         return hostnames
 
+    def get_recipset_group(self, taskspec):
+        for (jid, rset) in self.job_to_recipe_set_map.iteritems():
+            if taskspec == rset:
```

Comment:
I think the values in `self.job_to_recipe_set_map` are lists, not single values. If so, then the comparison would fail. Can you verify this?

Reply:
A valid concern. Verified, fixed.